module/settings.mjs

```diff
@@ -129,6 +129,28 @@ export default function registerSystemSettings() {
             calculate: "ED.Settings.Encumbrance.encumbranceWithEffects"
           }
     } );
+
+     /* -------------------------------------------------------------------------------- */
+    /*                                  GM Chat Avatar                                     */
+    /* -------------------------------------------------------------------------------- */
+    // Chat Avatar settings Header
+    game.settings.register( "ed4e", "chatAvatarHeader", {
+        name: "ED.Settings.Chat.chatAvatarHeader",
+        config: true,
+    } );
+    // Chat Avater Options
+    game.settings.register( "ed4e", "chatAvatar", {
+        name: "ED.Settings.Chat.chatAvatar",
+        hint: "ED.Settings.Chat.chatAvatarHint",
```

Comment:
Maybe `scope: "client"`? So players also can decide which way they want it.
See comments in chat.mjs